rest/model/dns/record.go

```diff
@@ -30,10 +30,10 @@ type Record struct {
 	Regions data.Regions `json:"regions,omitempty"`
 
 	// Contains the key/value tag information associated to the record
-	Tags map[string]string `json:"tags,omitempty"` // Only relevant for DDI
```

Comment:
These tags are actually omitted on the zones CRUD routes, unless explicitly instantiated.